pinc/CharSuites.inc

```diff
@@ -80,7 +80,6 @@ class CharSuite
                 ' on line ' . $trace[0]['line'],
                 E_USER_NOTICE
             );
-            return null;
```

Comment:
We need to edit the comment in line 75, probably just remove `and return null`

Reply:
Good spot.